docs/iris/src/whatsnew/1.8.rst

```diff
@@ -60,6 +60,9 @@ Iris 1.8 features
         air_potential_temperature / (K)     (model_level_number: 10; grid_latitude: 83; grid_longitude: 83)
     
 
+* The :class:`iris.analysis.Linear` scheme now supports regridding as well as interpolation.
+  This enables :meth:`iris.cube.Cube.regrid` to perform bilinear regridding, which now
+  replaces the experimental routine  "iris.experimental.regrid.regrid_bilinear_rectilinear_src_and_grid".
```

Comment:
@pp-mo Could you please make this the last entry in this section, rather than the first, thanks.